File: _01_GreenTradeEpt/BackEndCode/greenTradeEpt/greenTradeEpt_main/src/main/java/com/etlpat/service/ExpertService.java
package com.etlpat.service;

import com.etlpat.pojo.Expert;
import com.baomidou.mybatisplus.extension.service.IService;
import com.etlpat.pojo.PageBean;

/**
 * @author lenovo
 * @description 针对表【tb_expert】的数据库操作Service
 * @createDate 2025-06-10 12:19:58
 */
public interface ExpertService extends IService<Expert> {
    public PageBean<Expert> getPageAll(Integer pageNum, Integer pageSize);
}

File: _01_GreenTradeEpt/FrontEndCode/green-trade-ept-frontend-vue/src/apis/ExpertService.js
// 二次封装--针对业务逻辑的axios封装
import newAxios from "../util/AxiosUtils";

// 分页查询的请求
export async function getPageAll(pageNum, pageSize) {
  return (
    await newAxios.get("/expert/getPageAll/" + pageNum, {
      params: { pageSize }, // 请求参数
    })
  ).data;
}
